schedule/yast/ext4/ext4@yast-s390x-disk-activation.yaml

```diff
@@ -27,7 +27,7 @@ schedule:
   - installation/authentication/use_same_password_for_root
   - installation/authentication/default_user_simple_pwd
   - installation/resolve_dependency_issues
-  - installation/installation_overview
+  - installation/installation_settings/validate_default_target
```

Comment:
https://openqa.suse.de/tests/7923906#step/installation_overview/12

Reply:
Added validation for ssh service and the module for opening a port.